pyproject.toml

```
@@ -39,3 +39,39 @@ skip_glob = [
     "mypyc/test-data/*",
     "test-data/*",
 ]
+
+[tool.ruff]
+line-length = 99
```

you may want to also set

https://beta.ruff.rs/docs/settings/#target-version
```suggestion
target-version = "py37"
line-length = 99
```